sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/BUILD

```diff
@@ -117,13 +117,13 @@ opentitan_test(
     name = "cp_provision",
     cw310 = cw310_jtag_params(
```

Comment:
Just an FYI: the "fpga_cw310_sival" environment runs on hyper310, so whenever you add it, you'll want to make sure to assign it to an arg carrying `hyper310_jtag_params`, not `cw310_jtag_params`.